src/main/kotlin/no/nav/familie/ks/sak/api/ekstern/RestFerdigstillOppgaveKnyttJournalpost.kt

```diff
@@ -0,0 +1,17 @@
+import no.nav.familie.ks.sak.api.ekstern.RestJournalFøring
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingKategori
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingType
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingÅrsak
+import java.time.LocalDateTime
+
+data class RestFerdigstillOppgaveKnyttJournalpost(
```

Comment:
Ingen stor sak, men foretrekker som regel `Dto` (**D**ata **T**ransfer **O**bject) suffixen over `Rest` prefixen da den ikke er knyttet opp mot HTTP rest arkitekturen. Hvis man på et eller annet tidspunkt skulle bytte over til en annen teknologi trenger man ikke endre navnet. Dere får vurdere hva dere liker best. 😄 

```suggestion
data class FerdigstillOppgaveKnyttJournalpostDto(
```

Om dere bestemmer dere for å endre kan dere ta `RestJournalFøring` under også.